### DISCUSSION TITLE
[Feature Request] Don’t open comments/playlist queue upon fullscreen

### DISCUSSION THREAD

**Discussion by Hycade** [qnblackcat/uYouPlus#542] [Ideas]
If this is possible I would love to have the option to keep the comment/playlist queue UI closed when full-screening. The swipe to dismiss gesture is quite helpful but this would probably be better.

**Comment by arichornlover** [comment]
I’ve been trying to keep those closed on my phone, this idea is definitely good

**Comment by level3tjg** [comment]
There's an A/B test for this; `-[YTColdConfig isLandscapeEngagementPanelEnabled]`
